cmake/toolchains/cmake_aarch64_cross.toolchain

```diff
@@ -0,0 +1,55 @@
+#
```

Comment:
This file is not needed - it's for cross compilation of aarch64 ubuntu on x86 ubuntu, a config that has been made obsolete by native aarch64 ubuntu builds.